permissions/plugin-veracode-scan.yml

```
@@ -8,3 +8,5 @@ paths:
 developers:
   - "dennisgu"
   - "sbuhary"
+  - "sdhanaitveracode"
+  - "KetgalePranav"
```

I think that this needs to be the Jira account, not the GitHub account
```suggestion
  - "pranavketgale"
```